CsCore/CsCoreUnity/Plugins/CsCoreUnity/com/csutil/logging/test.cs

```diff
@@ -0,0 +1,45 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+public class SystemConsoleToUnityLogRedirector { 
```

Comment:
Please add docu what its purpose and give the test.cs class a better name. In general since this is in the CsCoreUnity folder I dont think it is needed at all, in Unity the normal usage of the Log class should always work also in the context of tests, just dont use Console. anywhere and always instead Log.